test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsFileParsingTests/CredentialsItemTests.cs

```diff
@@ -13,6 +13,10 @@ namespace NuGet.Configuration.Test
 {
     public class CredentialsItemTests
     {
+        private static readonly string _moreThanOneUsername = "A credentials item must only have one Username entry.";
```

Comment:
nit: 

You can actuall refer to the strings from the assembly you are testing. 
That way string changes don't require test changes.